oio/xcute/common/job.py

```diff
@@ -57,8 +57,11 @@ def sanitize_params(self, job_config):
         self.tasks_batch_size = int_value(
             job_config.get('tasks_batch_size'), None)
         if self.tasks_batch_size is None:
-            self.tasks_batch_size = min(
-                self.tasks_per_second * 2, self.MAX_TASKS_BATCH_SIZE)
+            if self.tasks_per_second > 0:
+                self.tasks_batch_size = min(
+                    self.tasks_per_second * 2, self.MAX_TASKS_BATCH_SIZE)
+            else:
```

Comment:
That should return an error?

Reply:
When `tasks_per_second` is less than or equal to 0, it means we want to go as fast as possible.